pylabrobot/resources/rotation.py

```diff
@@ -1,36 +1,47 @@
 import math
-
+from pylabrobot.resources.coordinate import Coordinate
 from pylabrobot.utils.linalg import matrix_multiply_3x3
 
-
 class Rotation:
   """ Represents a 3D rotation. """
 
   def __init__(self, x: float = 0, y: float = 0, z: float = 0):
-    self.x = x  # around x-axis, roll
-    self.y = y  # around y-axis, pitch
-    self.z = z  # around z-axis, yaw
+    self.x = x 
+    self.y = y
+    self.z = z
 
-  def get_rotation_matrix(self):
-    # Create rotation matrices for each axis
-    Rz = ([
+  def get_rotation_and_translation(self, origin: Coordinate = Coordinate.zero()):
+    # rotation matrices for each axis
```

Comment:
I was thinking about storing the origin as an attribute on the class itself

Reply:
good idea